lib/srv/db/db_test.go

```diff
@@ -427,13 +427,13 @@ func setupTestContext(ctx context.Context, t *testing.T) *testContext {
 	// Auth client/authorizer for database service.
 	dbAuthClient, err := tlsServer.NewClient(auth.TestServerID(teleport.RoleDatabase, hostID))
 	require.NoError(t, err)
-	dbAuthorizer, err := auth.NewAuthorizer(dbAuthClient, dbAuthClient, dbAuthClient)
+	dbAuthorizer, err := auth.NewAuthorizer("cluster-name", dbAuthClient, dbAuthClient, dbAuthClient)
```

Comment:
Can you use clusterName variable here and below instead of string constants.

Reply:
Done.